### ISSUE TITLE
Inconsistent behavior: url may not start to load at all

### ISSUE THREAD

**Issue by BenoitDuffez** [koush/ion#779]
I am downloading an image to an ImageView with this code:

```
Ion
    .with(this) // AppCompactActivity
    .load(image.getFullImageUrl())
    .progressBar(mProgressBar)
    .group(mMaximizedImage)
    .intoImageView(mMaximizedImage)
    .setCallback(new FutureCallback<ImageView>() {
        @Override
        public void onCompleted(Exception e, ImageView result) {
            L.d("imgpreview[" + position + "]", "end full image download " + image.getFullImageUrl() + ", error?: " + e);
            mMaximizedImage.setVisibility(View.VISIBLE);
            disableProgressBar();
        }
    });
```

Here's the result of two attempts: the first one at _21:25:22.155_ and the second one at _21:25:28.751_:
http://pastebin.com/eNc925b0

As you can see:
* the first attempt doesn't trigger anything in Ion, however the second one works as expected.
* the first Ion log happens at _21:25:28.764_, 13ms after the start (global Ion log tag: `picion`)
* the logging is set at the maximum verbosity.
* before downloading the image, I cancel any pending request, so we see a `CancellationException` happening.

I don't know what I can do to troubleshoot this, as repeating exactly the same operation yields different results.

I am using `compile 'com.koushikdutta.ion:ion:2.+'`.

**Comment by BenoitDuffez** [comment]
My cat closed the issue. I reopened it because the cat didn't fix the issue.

**Comment by koush** [comment]
Are you using Google Play Services in your app or no? What is the device you are testing on?

**Comment by BenoitDuffez** [comment]
Yes, firebase analytics. Tried to remove them already, (gms _and_ firebase), same behavior. Nexus 6P running the latest 7.0 stock.

**Comment by koush** [comment]
ok will check

**Comment by BenoitDuffez** [comment]
Thanks. Here's the app Gradle build script: http://pastebin.com/DXbxGWk3
The project build script just has the repo definitions for some dependencies/plugins.
Tell me if you need more information.

**Comment by BenoitDuffez** [comment]
Is there anything I can do to help?

**Comment by BenoitDuffez** [comment]
Hi, small ping here.
Also, same behavior with Android 7.1 dev preview.
Please tell me if you want me to do better troubleshooting to help fix this.
Thanks
